src/app/folder/folderlist.component.html

```diff
@@ -30,9 +34,13 @@
         <mat-icon *ngIf="node.folderType=='templates'" mat-list-icon class="folderIconStandard">description</mat-icon>
         <mat-icon *ngIf="node.folderType=='trash'" mat-list-icon class="folderIconStandard">delete</mat-icon>
         <mat-icon *ngIf="node.folderType=='user'" mat-list-icon class="folderIconUser">folder</mat-icon>
-	<span style="margin-left: 5px;">{{node.folderName}}</span>
+        <div style="margin-left: 5px;" 
+              draggable="true"
+              (dragstart)="dragFolderStart($event, node.folderId)">
+              {{node.folderName}}
+        </div>
         <span *ngIf="node.newMessages" matBadge="{{node.newMessages}}" matBadgeOverlap="true" class="newMessagesCount" matBadgeSize="medium">&nbsp;</span>
-	<span style="flex-grow: 1"></span>
+	      <span style="flex-grow: 1"></span>
```

Comment:
Extra whitespace? :­)

Reply:
yes, previous revision only had 1 tab on this line, while the ident level in that block is 4 tabs.